test/mock_server_state.hpp

```diff
@@ -189,6 +189,19 @@ namespace wsrep
             }
         }
 
+        int do_crypt(void**                ctx         WSREP_UNUSED,
+                     wsrep::const_buffer&  key         WSREP_UNUSED,
+                     unsigned int          key_version WSREP_UNUSED,
+                     const char            (*iv)[32]   WSREP_UNUSED,
+                     wsrep::const_buffer&  input       WSREP_UNUSED,
+                     void*                 output      WSREP_UNUSED,
+                     bool                  encrypt     WSREP_UNUSED,
+                     bool                  last        WSREP_UNUSED)
+            WSREP_OVERRIDE
+        { 
+            return 0;
```

Comment:
Returning 0 here may be confusing and lead to endless loop. I'd suggest either
 * to do a simple `memcpy()` from input to output and return the number of bytes copied
or
* return -ENOSYS or something like that